cdbbulksearch.py

```diff
@@ -189,7 +201,7 @@ def get(self):
                 future.add_done_callback(taskCounter.dec)
                 tasks.append(
                     (
-                        epd,
+                        epds,
```

Comment:
I don't think it is a good idea to store the full epds there, this could be a very large set, and could use quite some memory. Probably just the epd + the len would do.

Reply:
or is this just a ref in the python sense.. hm, that could be OK.

Reply:
Yep, just a reference.

Reply:
And it avoids the duplicate info from before (epd and epdIdx separately), too.